src/containers/StudyCreateForm/StudyCreateForm.tsx

```diff
@@ -64,97 +53,40 @@ const radioValues = [
     label: '오프라인',
   },
 ];
-const locations = [
-  {
-    value: 'SEOUL',
-    label: '서울',
-  },
-  {
-    value: 'BUSAN',
-    label: '부산',
-  },
-  {
-    value: 'DAEGU',
-    label: '대구',
-  },
-  {
-    value: 'INCHEON',
-    label: '인천',
-  },
-  {
-    value: 'GWANGJU',
-    label: '광주',
-  },
-  {
-    value: 'DAEJEON',
-    label: '대전',
-  },
-  {
-    value: 'ULSAN',
-    label: '울산',
-  },
-  {
-    value: 'JEJU',
-    label: '제주',
-  },
-];
 
-const people = [
-  {
-    value: 1,
-    label: '1',
-  },
-  {
-    value: 2,
-    label: '2',
-  },
-  {
-    value: 3,
-    label: '3',
-  },
-  {
-    value: 4,
-    label: '4',
-  },
-  {
-    value: 5,
-    label: '5',
-  },
-  {
-    value: 6,
-    label: '6',
-  },
-  {
-    value: 7,
-    label: '7',
-  },
-  {
-    value: 8,
-    label: '8',
-  },
-  {
-    value: 9,
-    label: '9',
-  },
-  {
-    value: 10,
-    label: '10',
-  },
-];
+const CreateSchema = Yup.object({
+  title: Yup.string()
+    .trim('앞, 뒤 공백을 제거해주세요.')
+    .strict()
+    .max(50, '50자를 넘을 수 없습니다.')
+    .min(5, '최소 5자 이상입니다.')
+    .required('제목을 입력해주세요.'),
+  description: Yup.string()
+    .trim()
+    .max(1000, '1,000자를 넘을 수 없습니다.')
+    .min(10, '최소 10자 이상입니다.')
+    .required('스터디 내용을 입력해주세요.'),
+});
 
 function StudyCreateFormContainer() {
-  const [title, setTitle] = useState('');
-  const [category, setCategory] = useState<string | number>('');
-  const [location, setLocation] = useState<string | number>('');
-  const [peopleNumber, setPeopleNumber] = useState<string | number>(0);
-  const [isOnline, setIsOnline] = useState(true);
-  const [startDate, setStartDate] = useState<string | number>('');
-  const [endDate, setEndDate] = useState<string | number>('');
+  const [topic, setTopic] = useState<string>('');
+  const [recruitsNumber, setRecruitsNumber] = useState<string>('');
+  const [region, setRegion] = useState<string>('');
+  const [isOnline, setIsOnline] = useState<boolean>(true);
+  const [startDate, setStartDate] = useState<string>('');
+  const [endDate, setEndDate] = useState<string>('');
   const [mbtiPreference, setMbtiPreference] = useState(true);
   const [mbtiCheckedList, setMbtiCheckedList] = useState<string[]>([]);
   const [imageSrc, setImageSrc] = useState<File | null>(null);
   const [thumbnailImage, setThumbnailImage] = useState<string>('');
-  const [body, setBody] = useState('');
+  const [fileErrorMessage, setFileErrorMessage] = useState<string>('');
+  const [topicErrorMessage, setTopicErrorMessage] = useState<string>('');
+  const [peopleErrorMessage, setPeopleErrorMessage] = useState<string>('');
```

Comment:
p4;
Error Message를 관리하는 State를 1개로 줄여보는 것은 어떨까요?
```suggestion
  const [ErrorMessage, setErrorMessage] = useState<object>({file: '', topic:'', people: ''});
```

Reply:
제가 코드를 잘 짰으면 괜찮았을 수 있는데, error message를 하나의 객체 state로 관리하면
```js 
const checkSelectOptions = () => {
    if (!topic) {
      setTopicErrorMessage('분야를 선택해주세요');
      setErrorMessages({ ...errorMessages, topic: '분야를 선택해주세요'});
    }
    if (!recruitsNumber) {
      setPeopleErrorMessage('인원수를 선택해주세요');
      setErrorMessages({ ...errorMessages, people: '인원수를 선택해주세요'});
    }
}
```
이 함수에서 topic에 대한 state가 변경이 되지 않습니다ㅜㅜ.. 저도 정확히는 모르는데 useState의 setState 반영이 우리가 생각하는 만큼 바로 반영되지 않는 것 같더라구요... 이부분에 대해서는 조금 더 알아볼 필요가 있습니다.. 이러한 이유로 우선 변경 없이 남겨두었습니다..! 코멘트 감사합니다 팽님👍